ShortURLBuilderProvider/Sources/ShortURLBuilderProvider/Services/Bitly/BitlyEndpoint.swift

```diff
@@ -11,7 +11,7 @@ import ShortURLNetwork
 enum BitlyEndpoint: BaseEndpoint {
     case shorten(String)
 
-    var baseURL: String { "https://api-ssl.bitly.com" }
+    var baseURL: String { "https://api-ssl.bitly.com" }  // "https://api-ssl.bitly.com"
```

Comment:
```suggestion
    /// https://api-ssl.bitly.com
    var baseURL: String { "https://api-ssl.bitly.com" }
```
let's use DocC if it's the intention, otherwise feel free to resolve the thread